libs/test-utils/src/compressed_tallies.ts

```diff
@@ -15,7 +15,10 @@ export function getZeroCompressedTally(election: Election): CompressedTally {
       return [0, 0, 0, 0, 0, 0, 0, 0, 0];
     }
     if (contest.type === 'candidate') {
-      return [0, 0, 0, 0, ...contest.candidates.map(() => 0)];
+      if (contest.allowWriteIns) {
+        return [0, 0, 0, 0, ...contest.candidates.map(() => 0)];
+      }
+      return [0, 0, 0, ...contest.candidates.map(() => 0)];
```

Comment:
Ultimately, nothing goes wrong if you just give an extra `0` for non write-in cases, but I changed this while debugging another issue and it makes sense.